pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/NamespacesTest.java

```diff
@@ -1828,10 +1830,18 @@ public void testSplitBundleForMultiTimes() throws Exception{
         String namespace = BrokerTestUtil.newUniqueName(this.testTenant + "/namespace");
         BundlesData data = BundlesData.builder().numBundles(4).build();
         admin.namespaces().createNamespace(namespace, data);
+        LookupOptions optionsHttps = LookupOptions.builder().authoritative(false).requestHttps(true).readOnly(false).build();
+        URL localWebServiceUrl = new URL(pulsar.getSafeWebServiceAddress());
         for (int i = 0; i < 10; i ++) {
             final BundlesData bundles = admin.namespaces().getBundles(namespace);
             final String bundle = bundles.getBoundaries().get(0) + "_" + bundles.getBoundaries().get(1);
+            NamespaceBundle testBundle = new NamespaceBundle(NamespaceName.get(namespace),
+                    Range.range(Long.decode(bundles.getBoundaries().get(0)), BoundType.CLOSED,
+                            Long.decode(bundles.getBoundaries().get(1)), BoundType.OPEN),
+                    pulsar.getNamespaceService().getNamespaceBundleFactory());
+            doReturn(Optional.of(localWebServiceUrl)).when(nsSvc).getWebServiceUrl(testBundle, optionsHttps);
```

Comment:
Can change to `doReturn(Optional.of(localWebServiceUrl)).when(nsSvc).getWebServiceUrl(testBundle, any());` since we don't care about the lookup options in this case?

Reply:
I tries, the test will fail